Modules/_testcapimodule.c

```diff
@@ -4809,10 +4817,12 @@ static PyMethodDef TestMethods[] = {
     {"getargs_y",               getargs_y,                       METH_VARARGS},
     {"getargs_y_star",          getargs_y_star,                  METH_VARARGS},
     {"getargs_y_hash",          getargs_y_hash,                  METH_VARARGS},
+// #if USE_UNICODE_WCHAR_CACHE
```

Comment:
I assume these are still needed somehow? Are they difficult to remove or just still needed by some tests?

Reply:
Remnants of debugging.